### PR TITLE
redirect to login page when unauthenticated (issue #83)

### PR DESCRIPTION
**Pull request by neckberg**

### CODE REVIEW

**Comment by carlschroedl** [comment]
Or we could have one test assert 302 when middleware is enabled but assert 200 when it is disabled.

File: routes/web.php [hunk 1]
```diff
@@ -11,41 +11,49 @@
 |
 */
 
-Route::get('/', function () {
-  return view('myElections');
-});
-
 Auth::routes();
 
 Route::get('/profile', 'ProfileController@index');
```
Comment (tylerharter) [inline routes/web.php]: Maybe comment that the auth framework already redirects these for us.

File: routes/web.php [hunk 2]
```diff
@@ -11,41 +11,49 @@
...
+  Route::get('/candidates/{election}', function ($election) {
+      return view('candidates', ['election' => $election]);
+  });
+  Route::get('/electorate/{election}', function ($election) {
+      return view('electorate', ['election' => $election]);
+  });
+  Route::get('/results/{election}', function ($election) {
+      return view('results', ['election' => $election]);
+  });
 });
```
Comment (tylerharter) [inline routes/web.php]: Why isn't /ballotReview/{election} inside this?

**Comment by tylerharter** [comment]
Regarding the test failures: I haven't looked at it, but I'm guessing it's just trying to see that some route is reachable?  Maybe the fix is as easy as making the test target a route that is not affected by the middleware.  E.g., maybe it could test that the verify_email route is reachable?

**Pull request merged**